Apps/Sandcastle/gallery/Custom Geocoder.html

```diff
@@ -53,8 +53,11 @@
 OpenStreetMapNominatimGeocoder.prototype.geocode = function (input) {
     var endpoint = 'https://nominatim.openstreetmap.org/search?';
     var query = 'format=json&q=' + input;
-    var requestString = endpoint + query;
-    return Cesium.loadJson(requestString)
+    var resource = new Cesium.Resource({
+        url: endpoint + query
```

Comment:
Shouldn't we use Resource properly and pass in the query parameters separately instead of globbing them together?